packages/sheet-permission-ui/package.json

```diff
@@ -0,0 +1,112 @@
+{
+    "name": "@univerjs/sheets-permission-ui",
```

Comment:
`sheets-protection-ui`. The package's name should be similar to its underlying plugin.